### PR TITLE
Add comparison to other packages

### PR DESCRIPTION
**Pull request by albertpod**
This PR addresses #71

### CODE REVIEW

File: docs/src/manuals/comparison.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+| **RxInfer.jl**             | –            | ✓          | ✓              | –                         | –          | Message-passing       | Julia    | –                     |
+| **Turing.jl**              | ✓            | –          | ✓              | –                         | –          | Sampling              | Julia    | ✓                     |
+| **PyMC**                   | ✓            | –          | ✓              | ✓                         | –          | Sampling              | Python   | ✓                     |
+| **Numpyro**                | ✓            | ✓          | –              | ✓                         | –          | Sampling              | Python   | ✓                     |
+| **TensorFlow Probability** | ✓            | –          | –              | ✓                         | –          | Sampling              | Python   | ✓                     |
+| **Infer.net**              | –            | ✓          | –              | ✓                         | –          | Message-passing       | C#       | –                     |
+
```
Comment (bvdmitri) [inline docs/src/manuals/comparison.md:19]: Can we put a date of the comparison?

File: docs/src/manuals/comparison.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+There's a plethora of probabilistic programming languages and packages available today. While all pivot around Bayesian Inference, their methodologies vary. This section juxtaposes `RxInfer.jl` against other renowned probabilistic programming languages and packages. The goal is to enlighten potential users about the nuances and guide them in choosing the package that resonates best with their requirements.
+
+**DISCLAIMER**: 
+1. This comparison isn't exhaustive and mirrors the author's hands-on experience with the packages. Some might have undergone more rigorous testing than others. If you're an author of one of these packages and believe the comparison doesn't do justice, please reach out, and we'll be more than willing to rectify.
+2. The comparison is more qualitative than quantitative, considering the intricacies of upkeeping benchmarking code for perpetually evolving packages.
+
+
+
+| Toolbox                    | Universality | Efficiency | Expressiveness | Debugging & Visualization | Modularity | Inference Engine      | Language | Community & Ecosystem |
```
Comment (bvdmitri) [inline docs/src/manuals/comparison.md:11]: PGmax, ForneyLab?, Stan

**Comment by albertpod** [comment]
@HoangMHNguyen made a few changes. Something occurred to me. Instead of binary table, perhaps we can make a ternary table.

For example, RxInfer is not universal compared to Turing, but it's more universal compared to ForneyLab or PGMax.

**Comment by HoangMHNguyen** [comment]
I think that's a good idea, but we can just write an extra note below the table instead of creating a new table.

**Comment by albertpod** [comment]
> @albertpod I think that's a good idea, but we can just write an extra note below the table instead of creating a new table.

@HoangMHNguyen perhaps we can use `~` symbol to denote partial capability?
Something like:
`✓` : Full capability or feature is present.
`~` : Partial capability or feature is present.
 `✗` : No capability or feature.

**Comment by bvdmitri** [comment]
That is the status of this PR? Can it be reviewed and merged potentially? (Sorry if I missed smth, just trying to catch up after my sickness)

**Comment by bvdmitri** [comment]
@HoangMHNguyen @albertpod is this ready?

**Comment by albertpod** [comment]
@bvdmitri yes

**Comment by bvdmitri** [comment]
Great, I skimmed through it, looks good to me! Thanks guys!!

**Pull request merged**